Source/Charts/Charts/BarLineChartViewBase.swift

```diff
@@ -931,7 +931,7 @@ open class BarLineChartViewBase: ChartViewBase, BarLineScatterCandleBubbleChartD
             ))
         {
             var scrollView = self.superview
-            while (scrollView !== nil && !scrollView!.isKind(of: NSUIScrollView.self))
+            while (scrollView !== nil && !(scrollView?.isKind(of: NSUIScrollView.self) ?? false))
```

Comment:
Same issue here where if the last component is evaluated and `scrollView == nil` we return `true`
Easiest way to solve this is to replace
`(scrollView !== nil && !(scrollView?.isKind(of: NSUIScrollView.self) ?? false))`
with
`!(scrollView?.isKind(of: NSUIScrollView.self) ?? true)`

Reply:
done